platforms/hyperledger-besu/charts/tessera_key_mgmt/templates/job.yaml

```diff
@@ -82,7 +84,39 @@ spec:
                 -d '{"role":"'"${VAULT_APP_ROLE}"'","jwt":"'"${KUBE_SA_TOKEN}"'"}' | \
                 jq -r 'if .errors then . else .auth.client_token end')
               validateVaultResponse 'vault login token' "${VAULT_CLIENT_TOKEN}"
+
+              echo $VAULT_CLIENT_TOKEN > /secret/VAULT_CLIENT_TOKEN
+         volumeMounts:
+         - name: vault-token
+           mountPath: /secret
+      containers:
+        - name:  "tessera-crypto"
```

Comment:
Q: What is this container for?

Reply:
`tessera-crypto` container is for generating the tessera key and pushing them to vault(using `tessera -keygen`), while the `fetch-vault-token` get the vault client token and passes it to `tessera-crypto`. This removed the dependency of having the utility tools in the tessera container image.